UnitTests/UnitTests.NetCore/Mvvm/Test_INotifyPropertyChangedAttribute.cs

```diff
@@ -75,5 +76,56 @@ public void Test_INotifyPropertyChanged_WithoutHelpers()
         public partial class SampleModelWithoutHelpers
         {
         }
+
+        [TestCategory("Mvvm")]
+        [TestMethod]
+        public void Test_INotifyPropertyChanged_WithGeneratedProperties()
+        {
+            Assert.IsTrue(typeof(INotifyPropertyChanged).IsAssignableFrom(typeof(SampleModelWithINPCAndObservableProperties)));
+            Assert.IsFalse(typeof(INotifyPropertyChanging).IsAssignableFrom(typeof(SampleModelWithINPCAndObservableProperties)));
+
+            SampleModelWithINPCAndObservableProperties model = new();
+            List<PropertyChangedEventArgs> eventArgs = new();
+
+            model.PropertyChanged += (s, e) => eventArgs.Add(e);
+
+            model.X = 42;
+            model.Y = 66;
+
+            Assert.AreEqual(eventArgs.Count, 2);
+            Assert.AreEqual(eventArgs[0].PropertyName, nameof(SampleModelWithINPCAndObservableProperties.X));
+            Assert.AreEqual(eventArgs[1].PropertyName, nameof(SampleModelWithINPCAndObservableProperties.Y));
+        }
+
+        // See https://github.com/CommunityToolkit/WindowsCommunityToolkit/issues/3665
```

Comment:
Did you mean to link to https://github.com/CommunityToolkit/WindowsCommunityToolkit/issues/4167 here?

Reply:
Whoops, bad copy-paste of that line 😄
Fixed in 5c1b35bd3fd45f34a1f626f5ea54c18d9354e1d8.